src/bind.js

```diff
@@ -18,8 +18,10 @@
  *
  * @return {Function}
  */
-function bind(callback) {
-  // write code here
+function bind(callback, ...args) {
+  return (...params) => {
```

Comment:
write it in one line
```suggestion
  return (...params) => callback(...args, ...params);
```